src/gsy_e/gsy_e_core/market_counters.py

```diff
@@ -48,6 +48,12 @@ def __init__(self):
             clearing_interval_min=ConstSettings.FutureMarketSettings.
             FUTURE_MARKET_CLEARING_INTERVAL_MINUTES)
 
+    def is_time_for_clearing(self, current_time: DateTime) -> bool:
+        """Only return if future markets are enabled."""
+        if ConstSettings.FutureMarketSettings.FUTURE_MARKET_DURATION_HOURS > 0:
```

Comment:
It seems that the unit tests don't cover a scenario where `FUTURE_MARKET_DURATION_HOURS <= 0`. If you think it's not that important feel free to skip it, otherwise it would be nice to cover this scenario as well. Thanks

Reply:
Makes sense, I added it here: [22b903e](https://github.com/gridsingularity/gsy-e/pull/1571/commits/22b903ed3f9e0110f00264f54fc2db707e7ba118)